### PR TITLE
Bridges of Bungee = {contract: SocketGateway, event: SocketBridge}

### PR DESCRIPTION
**Pull request by lequangphu**
## Thank you for contributing to Spellbook 🪄
Please open the PR in **draft** and mark as ready when you want to request a review. 

### Description:

Union bridges of Bungee across chains, add `amount_usd`.

---
quick links for more information:
- [README.md](https://github.com/duneanalytics/spellbook/blob/main/README.md)
- [spellbook docs](https://github.com/duneanalytics/spellbook/tree/main/docs)
- [CONTRIBUTING.md](https://github.com/duneanalytics/spellbook/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by lequangphu** [comment]
@jeff-dude , this pr includes my work on Lifi that you just merged. What should I do?

**Comment by jeff-dude** [comment]
> @jeff-dude , this pr includes my work on Lifi that you just merged. What should I do?

merging in main usually does the trick, looks like we're good now

File: dbt_subprojects/daily_spellbook/models/_projects/bungee/bungee_bridges.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}}
+
+{% set chains = [
+    'ethereum', 'zkevm', 'scroll', 'blast', 'linea', 'mantle', 'optimism',
+    'gnosis', 'arbitrum', 'zksync', 'base', 'bnb', 'polygon',
+    'avalanche_c', 'fantom'
+] %}
+
+{% for chain in chains %}
+    {{ bungee_SocketBridge(chain) }}
```
[the remark this answers is not in the document]
Comment (jeff-dude) [reply dbt_subprojects/daily_spellbook/models/_projects/bungee/bungee_bridges.sql:22]: you also won't have all the mapped tokens / pricing info, so i imagine this isn't what you want?

[the remark this answers is not in the document]
Comment (lequangphu) [reply dbt_subprojects/daily_spellbook/models/_projects/bungee/bungee_bridges.sql]: you are right, I fixed in the latest commit.

**Review by jeff-dude — APPROVED** [review]
nice, this should be all good now!

**Comment by lequangphu** [comment]
> nice, this should be all good now!

thanks!

**Pull request merged**